codespell_lib/_codespell.py

```diff
@@ -32,6 +32,8 @@
 # endpoint.  Emails are more restrictive, so the endpoint remains flexible.
 uri_regex_def = (u"(\\b(?:https?|[ts]?ftp|file|git|smb)://[^\\s]+(?=$|\\s)|"
                  u"\\b[\\w.%+-]+@[\\w.-]+\\b)")
+inline_ignore_regex = re.compile(
+    r"[^\w\s] codespell:ignore(?:(?:\s|$)([\w,]*))")
```

Comment:
Won't this also match, and hence ignore the typos in this:
`You could also use the line based ignor ( codespell:ignore ) to ignor the whole line.`

Which I'd call a false positive!

Reply:
Questionable what the example sentence you give here should mean.  False positives are always a problem but then it looks like you constructed the example to produce the false positive.  In other words your sentence is probably not something a user would type.   (In fact, you probably can only write down ` codespell:ignore ` in technical documentation *about* codespell itself.)  

For prose, "inline-disabling" is probably not what you want anyway because there are no comments in prose.  A user would read the technical ignore statement in any case.  (There is also no syntax highlighting in the editor which would gray out our ignore statements.)    

Usually, such ignore statements are the last thing on a line except you have other *comments* on the same line,  so maybe we can change the behavior and only let `... | codespell:ignore | ...` match t.i. on the left an right we find the same character, thus the ` ( ... ) ` you gave would not match.   I'll push an update here.

Actually a simple `... (codespell:ignore) ...` would be very nice syntax for prose and I wonder if a simpler `r"[^\w\s]\s?codespell:ignore\b(\s+(?P<words>[\w,]*))?"` would *just* work or "be enough" in practice.